gp-limit-choices/gplc-gspc-compatibility.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Gravity Perks // GP Limit Choices // GSPC compatibility
+ *
+ * This is snippet will ensure that validation is checked again before an order is created.
+ * https://gravitywiz.com/documentation/gravity-forms-limit-choices/
+ */
+
+add_action( 'woocommerce_checkout_create_order', function( $order ) {
+	if ( ! function_exists( 'gp_limit_choices' ) ) {
+		return;
+	}
+
+	$items = $order->get_items();
+
+	foreach ( $items as $item ) {
+		$gspc_order_item = new WC_Order_Item( $item );
```

Comment:
```suggestion
		$gspc_order_item = new GS_Product_Configurator\WC_Order_Item( $item );
```

I think you're going to need the namespace here. Is this snippet confirmed working right now?

Reply:
@claygriffiths  Yes the snippet is working.